libc.in

```diff
@@ -226,8 +228,14 @@ if $backup; then
 	fi
 fi
 
+# Protect against symlink attack, ensure new file does not exist
+rm -f "$resolv_conf_tmp" 2>/dev/null || :
```

Comment:
There is no need to trail this with `|| :` because -f will never return an error.

Reply:
I did not know that. It does not even print any error if file is missing. Fixed.